torch/csrc/jit/testing/module_differ.h

```diff
@@ -0,0 +1,32 @@
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/jit/mobile/module.h>
+
+namespace torch {
+namespace jit {
+
+// Compares 2 mobile::Module. Comparison is done as follows:
+// 1. _ivalue() returned by both should be equal according to ivalueEquals below
+// 2. all functions with same name shall have same instructions and constants
+// 3. all functions in lhs exists in rhs.
+TORCH_API bool moduleEquals(
```

Comment:
I assume it's used in serialization only, not for deserialization. Could you confirm?

Reply:
this function is used for manual testing. It's not used in either serialization or loading